src/signer.js

```diff
@@ -60,6 +63,9 @@ chrome.webRequest.onBeforeRequest.addListener(
 	  if (!enabled || !valid())
 		  return;
 
+    if (url_pattern && !details.url.match(url_pattern))
+      return;
```

Comment:
I tried to assign `url_pattern` to the `urls` option down below, but it did not work. So I had to do this match here.